virttest/qemu_devices/qdevices.py

```diff
@@ -1364,6 +1364,10 @@ def get_qmp_args(self):
             args["backend"]["data"] = {"device": self.get_param("path")}
             return args
 
+        if self.params.get("backend") in "ringbuf":
+            args["backend"]["data"] = {"size": self.get_param("size", int(2048))}
```

Comment:
Please keep each line within 79 characters, thanks.

Reply:
thanks and update.